### PR TITLE
Venafi - add limit arg to get certificates cmd

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/30705

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests - will add in a different PR
- [x] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/Venafi/Integrations/README.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    "Venafi": {
+        "Certificate": {
+            "CreatedOn": "2018-07-16T16:35:35.9468326Z",
+            "DN": "\\VED\\Policy\\Venafi Operational Certificates\\WIN-MLK71Q10559",
+            "ID": "2a25573b-745c-4018-806a-e5c73f424675",
+            "Name": "WIN-MLK71Q10559",
+            "ParentDN": "\\VED\\Policy\\Venafi Operational Certificates",
+            "SchemaClass": "X509 Server Certificate"
+        },
+        "Certificats": {
```
Comment (yaakovi) [inline Packs/Venafi/Integrations/README.md:96]: should we remove the deprecated context paths from the README?

File: Packs/Venafi/Integrations/README.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            "ParentDN": "\\VED\\Policy\\Venafi Operational Certificates",
+            "SchemaClass": "X509 Server Certificate"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Venafi certificats query response
```
Comment (yaakovi) [inline Packs/Venafi/Integrations/README.md:110]: should be fixed in the code as well
```suggestion
>### Venafi certificates query response
```

File: Packs/Venafi/Integrations/README.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                "ValidationState": "Failure"
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Venafi certificats details
```
Comment (yaakovi) [inline Packs/Venafi/Integrations/README.md:213]:
```suggestion
>### Venafi certificates details
```

**Review by yaakovi — APPROVED** [review]

File: Packs/Venafi/Integrations/README.md (hunk 3 above)
Comment (Itay4) [reply Packs/Venafi/Integrations/README.md]: done

File: Packs/Venafi/Integrations/README.md (hunk 2 above)
Comment (Itay4) [reply Packs/Venafi/Integrations/README.md]: done

File: Packs/Venafi/Integrations/README.md (hunk 1 above)
Comment (Itay4) [reply Packs/Venafi/Integrations/README.md]: removed

**Comment by ShirleyDenkberg** [comment]
@yaakovi @ShahafBenYakir @roysagi Doc review completed.

**Pull request merged**